docs/usage/visualize.ipynb

```diff
@@ -539,6 +533,66 @@
     "graphviz.Source(dot)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Collapsing the graph also works for custom properties:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "scroll-input"
```

Comment:
Could you also make this `hide-input`, like the cell with the mass rendering?

Reply:
See also https://github.com/ComPWA/qrules/pull/274#discussion_r1670741589

Reply:
The benefit would be that you can remove the text between the two cells and write e.g. "Or any other properties of a `State`, such as its mass or its $J^{PC}(I^G)$ numbers."
![image](https://github.com/ComPWA/qrules/assets/29308176/fc4177a7-4d5a-4ea9-9f26-0a84ae7ecd33)